creating_reference_files/Generic_Instructions_for_Creating_ReferenceFiles.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f79e8d05",
+   "metadata": {},
+   "source": [
+    "Generic Instructions for Creating  New Reference Files\n",
+    "=======================================\n",
+    "\n",
+    "Tutorial Approach\n",
+    "---------------------\n",
+    "\n",
+    "We recommend that you review the tutorial for creating Flat Field Reference files \n",
+    "before doing this one.\n",
+    "\n",
+    "Rather than create a tutorial for each specific kind of reference file, this\n",
+    "tutorial will show how to use either the schema for that reference file as a \n",
+    "guide, or a simplified representation of the schema (the latter should be much\n",
+    "easier, and such information will be added to this tutorial as new reference \n",
+    "file types are added.\n",
+    "\n",
+    "This tutorial will use a few examples of how to do this,\n",
+    "and we expect that it will illustrate how to do the same for any reference file type.\n",
+    "More than one example is necessary since no particular one addresses all the \n",
+    "various aspects one may encounter.\n",
+    "\n",
+    "First, common elements to all reference files. All reference files have a meta\n",
+    "attribute, so one may do the following for all cases"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2890d2b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import roman_datamodels.datamodels as rdd\n",
+    "import roman_datamodels.stnode as rds\n",
+    "import numpy as np\n",
+    "import asdf\n",
+    "from astropy.time import Time\n",
+    "# create meta dictionary and populate with common elements\n",
+    "meta = {'telescope': 'ROMAN', 'reftype': '<the expected value for this refernece file>'}\n",
+    "# Create the information about instrument configuration for this flat field.\n",
+    "# The first item is always the same, but the others depend on the particular flat field\n",
+    "instrument = {'name': 'WFI', 'detector': 'WFI01', 'optical_element': 'F129'}\n",
+    "meta['instrument'] = instrument\n",
+    "# All the rest of these must be provided by the person who has constructed the flat field\n",
+    "meta['pedigree'] = \"\"\"whatever text you wish to describe the pedigree\n",
+    "    and it may go on and on and on and on and on and on\n",
+    "    and on and on and on and on and on\n",
+    "    and on and on, to your heart's content\"\"\"\n",
```

Comment:
`GROUND`

Reply:
as for p_ attributes, certainly not for source attributes since INS is making these. Will these have catalog entries? If not, none of these need to be added. If they are going to extract info for the catalogs like pedigree and such, yes, those do need to be added. Who knows the answer to this?

Reply:
Let me ask Lisa

Reply:
I did, and she says none are needed for reference files.